python/observability-online-eval/smolagents/example.py

```
@@ -0,0 +1,104 @@
+    output = ""
+    with engine.connect() as con:
+        rows = con.execute(text(query))
+        for row in rows:
+            output += "\n" + str(row)
+    return output
+
+
+try:
+    # Preferred (per docs/examples)
+    from smolagents import InferenceClientModel as HFModel
+except Exception:  # pragma: no cover - fallback for older/newer versions
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)